test/module/shared_model/backend_proto/shared_proto_tx_response_test.cpp

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "backend/protobuf/transaction_responses/proto_tx_response.hpp"
+
+#include <gtest/gtest.h>
+
+// Quite hacky way to extract the class name
+// probably not stable and should be improved
+class ClassNameVisitor : public boost::static_visitor<const std::string> {
+ public:
+  template <typename T>
+  const std::string operator()(const T &t) const {
+    auto s = t->toString();
+    // skip everything except class name
+    return std::string(s.begin(), s.begin() + s.find(':'));
+  }
+};
+
+class ProtoTxResponse : public testing::Test {
+ public:
+  void SetUp() override {}
```

Comment:
No reasons to override method here, isn't it?

Reply:
There is, it inherited from testing::Test and most probably cause a warning (like missing override, or smth)

The method here because overloaded SetUp has different signature and conflicts with the original one

Reply:
>There is, it inherited from testing::Test and most probably cause a warning (like missing override, or smth)

Did you check it?

Reply:
Sure